packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -349,7 +356,30 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
     </TextInputGroup>
   );
 
-  const expandableToggle = (
+  const expandToggleButton = (
+    <Button
+      variant={ButtonVariant.plain}
+      aria-label={toggleAriaLabel}
+      aria-expanded={isExpanded}
+      icon={<SearchIcon />}
+      onClick={onExpandHandler}
+      ref={searchInputExpandableToggleRef}
+      {...(isExpanded && { inert: '' })}
```

Comment:
Just a comment, but if it's preferred, you could put `inert` on the `<InputGroupItem />` and it will render everything in it as inert. There is only one one thing in it now, so there's no benefit to changing it, but I would probably put `inert` on the `<InputGroupItem />` to indicate that the whole element should be ignored. Similar to if we conditionally rendered the expand toggle, I'd conditionally render the `<InputGroupItem />`, not just the button in it.